andela/baby-face-rc
# #155003790 Product category page

#### What does this PR do?
Create a product category page
#### Description of Task to be completed?
Users should be able to click a link from the landing page to view a particular category of products
#### How should this be manually tested?
- Navigate to https://baby-face-rc-staging.herokuapp.com/
- Click on a product category link
- You should see a list of products in that category

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
story type: feature
story Id: 155003790
story title: Product category page
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
